content/hi/includes/task-tutorial-prereqs.md

```diff
@@ -0,0 +1,3 @@
+आपको कुबरनेट्स क्लस्टर की ज़रूरत पड़ेगी और क्यूब सीटीएल कमांड लाइन साधन को समनुरूप करना होगा ताकि वो आपके क्लस्टर के साथ संवाद कर सके। ये अनुशंसित किया गया है की इस अनुशीक्षण को क्लस्टर में रन करने के लिए कम से कम दो नोड का इस्तेमाल करे जो के कंट्रोल प्लेन होस्ट के तरह ना एक्ट करे। अगर आपके पास पहले से क्लस्टर नही है, आप मिनिक्यूब की मदद से बना सकते है या आप नीचे दिए हुए इन दो कुबरनेट्स प्लेग्राउंड का इस्तेमाल कर सकते है :

```

Comment:
```suggestion
आपको कुबरनेट्स क्लस्टर की ज़रूरत पड़ेगी और क्यूब सीटीएल कमांड लाइन साधन को 
समनुरूप करना होगा ताकि वो आपके क्लस्टर के साथ संवाद कर सकें। हमारी सलाह है की इस टुटोरिअल को क्लस्टर में रन करने के लिए कम से कम दो नोड का इस्तेमाल करे जो कि कंट्रोल प्लेन 
होस्ट के तरह ना एक्ट करे। अगर आपके पास पहले से क्लस्टर नही है, आप 
[minikube](https://minikube.sigs.k8s.io/docs/tutorials/multi_node/)
की मदद से वह बना सकते है या आप नीचे दिए हुए इन दो कुबरनेट्स प्लेग्राउंड का इस्तेमाल कर सकते हैं:

```

Reply:
Feel free to use new line when needed, markdown doesn't mind and it makes it more easily readable